CAN-D/Src/can.c

```diff
@@ -55,18 +55,18 @@ void HAL_CAN_MspInit(CAN_HandleTypeDef* canHandle)
     HAL_GPIO_Init(GPIOB, &GPIO_InitStruct);
 
     /* CAN interrupt Init */
-    HAL_NVIC_SetPriority(USB_HP_CAN_TX_IRQn, 5, 0);
-    HAL_NVIC_EnableIRQ(USB_HP_CAN_TX_IRQn);
-    HAL_NVIC_SetPriority(USB_LP_CAN_RX0_IRQn, 5, 0);
-    HAL_NVIC_EnableIRQ(USB_LP_CAN_RX0_IRQn);
-    HAL_NVIC_SetPriority(CAN_RX1_IRQn, 5, 0);
+    // Disabling unused interrupts for now
+//    HAL_NVIC_SetPriority(USB_HP_CAN_TX_IRQn, 5, 0);
+//    HAL_NVIC_EnableIRQ(USB_HP_CAN_TX_IRQn);
+//    HAL_NVIC_SetPriority(USB_LP_CAN_RX0_IRQn, 5, 0);
+//    HAL_NVIC_EnableIRQ(USB_LP_CAN_RX0_IRQn);
+    HAL_NVIC_SetPriority(CAN_RX1_IRQn, 1, 0);

```

Comment:
Since we're only working with CAN for now, we can safely set its priority higher (lower number). When we can start testing on a board, we'll set priorities. All priorities right now are defaulted to 5.